huntsman/observatory.py

```diff
@@ -325,26 +326,22 @@ def take_dark_fields(self,
            in the targets list.
 
         Args:
-            exptimes_list (list): List of exposure times for darks
+            exptimes (list): List of exposure times for darks
             wait_interval (float, optional): Time in between dark exposures
```

Comment:
What are the units of `wait_interval`? Probably seconds - but good to be specific.